src/api_server/swagger/firecracker.yaml

```diff
@@ -699,6 +811,19 @@ definitions:
         description: The amount of milliseconds it takes for the bucket to refill.
         minimum: 0
 
+  Vm:
```

Comment:
Please add a commit on top of https://github.com/firecracker-microvm/firecracker/pull/1537 to update that proposal with the details in this PR.